### DISCUSSION TITLE
Thumbnails hosted on AWS?

### DISCUSSION THREAD

**Discussion by PerpSearch** [mrlt8/docker-wyze-bridge#1144] [Q&A] [answered]
To begin, I know nothing about coding, rather I'm more of an end user that occasionally pokes at stuff.  Today I was poking around and clicked on the little (i) icon in the upper right corner and it spooled down a bunch of information about the camera.  Some of it I could understand, but the parameters for the thumbnail appears to be hosted on Amazon ([https://camplus-usw2.s3.us-west-2.amazon...](https://camplus-usw2.s3.us-west-2.amazonaws.com. . . and a bunch of code).  I'm assuming this is unavoidable, but I was wondering if there might be a way not to have to store them there or if it's more a Wyze thing we just have to live with.
![thumbnail](https://github.com/mrlt8/docker-wyze-bridge/assets/92053831/618ff1f5-f49a-4a20-ba33-a566a262f4e0)

**Comment by markdza** [comment, marked as answer]
> it's more a Wyze thing we just have to live with

^ this
